cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/jmx/NodeRepairJob.java

```diff
@@ -83,15 +70,13 @@ public boolean repairNextKeyspace() {
             commandToKeyspace.put(commandNo, keyspace);
 
             LOGGER.info("Submitted repair for keyspace {} with cmd#{}", keyspace, commandNo);
-            return true;
+            break;
         }
     }
 
-    private void finished() {
+    protected void finished() {
```

Comment:
The method name `finished` to me sounds like it should return a boolean whether the job is finished or not.  If this is supposed to be the "method that is called when job is done" it should probably be renamed to `finish`.